tests/infer/mcmc/test_nuts.py

```diff
@@ -182,9 +182,7 @@ def model(data):
     assert_equal(samples["p_latent"].mean(0), true_probs, prec=0.02)
 
 
-@pytest.mark.parametrize("jit", [False, mark_jit(True, marks=[
-    pytest.mark.skipif("CUDA_TEST" in os.environ, reason="FIXME: Too small step size with CUDA on JIT.")
-])], ids=jit_idfn)
```

Comment:
this error is fixed in PyTorch before the last release.

Reply:
Thanks for cleaning this up!